crates/typed-store/src/rocks/mod.rs

```diff
@@ -1825,14 +1864,40 @@ pub fn base_db_options() -> DBOptions {
     }
 
     // The table cache is locked for updates and this determines the number
-    // of shards, ie 2^6. Increase in case of lock contentions.
+    // of shards, ie 2^10. Increase in case of lock contentions.
     opt.set_table_cache_num_shard_bits(10);
 
+    // Allow buffering 1GiB of writes before flushing to disk.
+    opt.set_write_buffer_size(256 * 1024 * 1024);
```

Comment:
This will not really buffer 1GiB of writes before flushing to disk. My understanding is `set_max_write_buffer_number` is the maximum number of write bufffers unflushed before rocksdb will start blocking writes. If we want to buffer 1Gib, I think we want to use `set_min_write_buffer_number_to_merge(4)`

Reply:
For `set_min_write_buffer_number_to_merge()`, do you know what will happen if db write buffer or wal size limit is reached, but the column family that should be picked for compaction does not have 4 write buffers? For now, I will put `set_min_write_buffer_number_to_merge()` into `optimize_for_write_throughput()` as before.

Btw since epoch tables are opened in transactional mode, flushed write buffers are kept in memory.